docker/Dockerfile

```diff
@@ -29,7 +30,7 @@ RUN echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" > /etc/a
 # RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1E9377A2BA9EF27F
 RUN apt-get update && apt-get install -y clang-8 lld-8
 
-RUN apt-get update && apt-get install -y cmake
+RUN apt-get update && apt-get install -y cmake texinfo
```

Comment:
Thanks, I haven't used the docker configuration lately. I guess texinfo is only needed for GDB?

Reply:
Yup, the texinfo is for GDB, you're right, it should be possible to see if there's a way to suppress the texinfo files.